### ISSUE TITLE
app: graph layout

### ISSUE THREAD

**Issue by simpsus** [simpsus/factoriograph#5]
the current graph looks like it is in portrait mode. 
We could layout it top-down and have the graph canvas take more horizontal space.

**Comment by amarvin** [comment]
Good idea. Dagre does top-down layout by default, so in `application.py` line 22 you can either remove the "rankDir" layout setting or set it to the default "TD".
